sdk/textanalytics/ai-text-analytics/src/generated/models/index.ts

```diff
@@ -107,6 +107,24 @@ export interface DocumentError {
   error: TextAnalyticsError;
 }
 
+/**
+ * An interface representing TextAnalyticsWarning.
+ */
+export interface TextAnalyticsWarning {
+  /**
+   * Error code. Possible values include: 'LongWordsInDocument', 'DocumentTruncated'
+   */
+  code: WarningCode;
+  /**
+   * Warning message.
+   */
+  message: string;
+  /**
+   * A JSON pointer reference indicating the target object.
```

Comment:
```suggestion
   * An RFC 6901 JSON pointer indicating the target object.
```

"Reference" seems a bit confusing, and seems good to call out the relevant spec.

Reply:
We're actually removing this property for now since it's a placeholder and no concrete warnings use it.